### DISCUSSION TITLE
content of package 'nc-help' [Sep 03]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6350] [Discord Threads]
> 🙋‍♂️ **bmnnnkls**

**Comment by o1lab** [comment]
> 🙋‍♂️ **bmnnnkls**  
#### Dear <@&682859083799199814>. i would be very grateful if you could give some information on this.  
#  
> 👨‍🚀 **wingkwong**  
#### nc-help is closed source.  
#  
> 👨‍🚀 **wingkwong**  
#### and we moved most of the code back to nocodb package.  
#  
> 🙋‍♂️ **bmnnnkls**  
#### ok thanks for the clarification  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1135715818513109064)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
